ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/MapFragment.java

```diff
@@ -76,10 +62,18 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     public void onMapReady(@NonNull GoogleMap googleMap) {
         MapsInitializer.initialize(getContext());
         googleMap.setMapType(GoogleMap.MAP_TYPE_NORMAL);
-        addMapMarkers(googleMap, cityList.subList(0, MAX_CITIES));
+        addMapMarkers(googleMap, reducedCityList());
         googleMap.moveCamera(CameraUpdateFactory.newLatLng(MAP_CENTER));
     }
 
+    private List<WeatherData> reducedCityList() {
```

Comment:
nit: you will always want to gear towards action when naming methods i.e. `getReducedCityList()`